resources/netcanv.desktop

```diff
@@ -2,7 +2,7 @@
 Type=Application
 Encoding=UTF-8
 Name=NetCanv
-Comment=Multiplayer Paint
+Comment=Online collaborative painting
```

Comment:
Likewise, the app description is the following:
https://github.com/liquidev/netcanv/blob/ec91593adaabb1956fb12a94af30b895f9600b4a/build/windows-icons.ps1#L55